src/libraries/System.Security.Cryptography.Cose/ref/System.Security.Cryptography.Cose.csproj

```diff
@@ -0,0 +1,28 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>$(NetCoreAppCurrent);$(NetCoreAppMinimum);netstandard2.0;$(NetFrameworkMinimum)</TargetFrameworks>
```

Comment:
For the ref I'm surprised it's not just netstandard2.0

Reply:
For System.Formtat.Cbor we list the same four tfms, but that project was on netcoreapp and was ported to ns2.0 **and includes APIs that are not available there**.
https://github.com/dotnet/runtime/blob/ca6716a658137b55d29b3ce100d5d14eb0edd67e/src/libraries/System.Formats.Cbor/ref/System.Formats.Cbor.csproj#L3

I can see this one being only ns2.0 just wanted to point that out.

Reply:
Looks like https://github.com/dotnet/runtime/blob/main/src/libraries/System.Formats.Asn1/ref/System.Formats.Asn1.csproj#L3 has the same.  I'm guessing this is so that a netfx-consumer (or netcoreapp-consumer) can skip the netstandard DLL type forwards for things like System.Object and just get straight at the one appropriate for their runtime.